FFmpegInterop/UncompressedVideoSampleProvider.cpp

```diff
@@ -151,7 +152,7 @@ IMediaStreamDescriptor^ UncompressedVideoSampleProvider::CreateStreamDescriptor(
 	return ref new VideoStreamDescriptor(videoProperties);
 }
 
-HRESULT UncompressedVideoSampleProvider::InitializeScalerIfRequired()
+HRESULT UncompressedVideoSampleProvider::InitializeScalerIfRequired(AVFrame* avFrame)
 {
```

Comment:
I made this change when we copied the hw frame to sw frame, in order to ensure that whatever the hw pixel format was, the sw frame was always the right output. I do not believe this is needed anymore, and can be reverted.